packages/account/src/Containers/Account/page-overlay-wrapper.tsx

```diff
@@ -44,12 +44,16 @@ const PageOverlayWrapper = observer(({ routes, subroutes }: PageOverlayWrapperPr
     }));
 
     const onClickClose = React.useCallback(() => {
+        if (is_from_tradershub_os) {
+            window.location.href = 'https://hub.deriv.com/tradershub/cfds';
```

Comment:
Can we have it based on env? 

If staging, it should go to [staging-hub](https://staging-hub.deriv.com/tradershub/cfds)
For production, 'https://hub.deriv.com/tradershub/cfds'

Reply:
For example: https://github.com/deriv-com/deriv-app/pull/17649/files#diff-b319cb2d37ec6cbe463bb9a37bec69d89f9bf65a8e8b9474940ba611f8c572eeR38